docs/Design/Exceptions/Exception Handling.md

```diff
@@ -0,0 +1,355 @@
+# Exception Handling
```

Comment:
Make sure you implement the requested changes from the last review.

https://github.com/HighSchoolHacking/GLS/pull/148#pullrequestreview-4365921